pkg/parser/ast.go

```diff
@@ -730,3 +730,11 @@ func zeroValue(t *Type, tt *lexer.Token) Node {
 	}
 	return nil
 }
+
+func isCompositeLit(n Node) bool {
+	if _, ok := n.(*ArrayLiteral); ok {
+		return true
+	}
+	_, ok := n.(*MapLiteral)
+	return ok
```

Comment:
ok. i thought about it and opted against it initially because it is not `, ok` anymore and it will always eval the cast to `*MapLiteral` short of being a smart compiler which Go surely is, but not 100% on tinygo .... but it's shorter and more obvious so I will change.

Done.